queue/python/queue.py

```diff
@@ -0,0 +1,46 @@
+# Implementation of queue using Linked List:
+
+class Node:
+    def __init__(self, data):
+       self.data = data
+       self.next = None
+ 
+class Queue:
+    def __init__(self):
+        self.head = None
+        self.last = None
+ 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp_mrnlsdu/queue/python/queue.py
+++ b/tmp/tmp_mrnlsdu/queue/python/queue.py
@@ -9,7 +9,7 @@
     def __init__(self):
         self.head = None
         self.last = None
- 
+
     def enqueue(self, data):
         if self.last is None:
             self.head = Node(data)
```